### DISCUSSION TITLE
Correct UI/UX inconsistencies relating to categories

### DISCUSSION THREAD

**Discussion by bzg** [codegouvfr/sill-web#465] [Ideas]
Les `catégories` apparaissent dans les filtres quand on cherche un logiciel du SILL.

Mais on ne sait pas d'où viennent ces catégories et on ne sait ni comment les ajouter au moment du référencement d'un logiciel ni comment les éditer sur les fiches existantes.

**Comment by garronej** [comment]
It's not great indeed.  
Thoses category are generated by chatGPT. I asked the SILL to split the softwares into categories.  
https://github.com/codegouvfr/sill-data/blob/1df0cde753a44c84678fe79be808dca584a599a7/software.json#L156-L158  

At the moment the categories can only be managed by editing manually je JSON files.  

When a software is added, by default, it has no category and thus only appear in the "all" filter.

**Comment by bzg** [comment]
Continuons en français sur les issues ouvertes en français et en anglais sur celles ouvertes en anglais (je vais clarifier ça dans un `CONTRIBUTING.md`).

ChatGPT ne doit être utilisé nulle part pour le SILL pour l'instant: ni pour générer du code ni pour générer des données.

> At the moment the categories can only be managed by editing manually je JSON files.

OK, discutons avec @JeromeBu des améliorations possibles via l'UI.

**Comment by guillermau** [comment]
Différence entre applicationCategory et keywords ?
Fusion entre applicationCategory de Software et SoftwareExternalData ?

**Comment by guillermau** [comment]
Affichage des champs catégories dans #128 
Besoin pour #129 car besoin de faire la liste des catégories pour la recherche

Création d'un champ modifiable dans le formulaire ?
